### DISCUSSION TITLE
Problem with io_random_placement

### DISCUSSION THREAD

**Discussion by salmen20** [The-OpenROAD-Project/OpenROAD-flow-scripts#1940] [General] [answered]
<img width="289" alt="ioplacement" src="https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/62488705/4cd18660-56e5-485b-96d3-05f5367f4beb">

i tried to fix it by increasing the DIE_AREA but still facing the same problem ,
if anyone knows how can i increase the die perimeter as i couldnt find any env variable for it 

thanks

**Comment by rovinski** [comment, marked as answer]
Increasing die area should fix the problem. Please try again.

**Reply by GSBhatta** [reply]
Should the Die Area be increased in the config.mk file under platforms directory or the one in our <top module name> directory?

**Reply by rovinski** [reply]
There shouldn't be a `DIE_AREA` in the platform `config.mk`. Die area is completely design dependent, so it should always be specified at the design config.mk level.

If you were using `CORE_UTILIZATION` instead of `DIE_AREA`, that is something which is more tied to the platform.
